### ISSUE TITLE
[QST] Support for movmatrix

### ISSUE THREAD

**Issue by carlguo866** [NVIDIA/cutlass#1696]
**What is your question?**

I would like to use [movmatrix.sync.aligned.shape.trans](https://docs.nvidia.com/cuda/parallel-thread-execution/index.html#warp-level-matrix-instructions-movmatrix) but there doesn't seem to be a primitive supporting `movmatrix` like `ldmatrix`. Is it just not supported yet by cutlass? Happy to do a PR for this as well

**Comment by thakkarV** [comment]
yeah we have not needed this yet. you are welcome to create an MR with the atom and appropriate tests added along with it
